index.html

```diff
@@ -202,47 +177,41 @@ <h2>Photo Gallery</h2>
         </section>
         <section class="survey">
         <h2>Survey</h2>
-         <form >
-         <label>What is your name ?:<br>
-         <input type="text" name="name_field"></label>
-         <br><br>
-         <label>What is your email ?:<br>
-          <input type="email" name="email_field"></label>
-          <br><br>
-          <label>Which integer is best ?:<br>
-        <input type="number" name="best_integer"></label>
-        <br><br>
-        <label>Which day is best ?:<br>
-        <input type="date" name="best_day"></label>
-        <br><br>
-        </form>
-        <label>Which bear is best ?:<br>
-        <input list="bears" name="best_bear" ></label>
-        <datalist id="bears">
-        <option value="none">
-        <option value="Black">
-        <option value=" Brown">
-        <option value="Care">
-        <option value="Panda">
-        <option value="Polar">
-        <option value="Teddy">
-        </datalist> 
-        <br><br>
-        <label>Do you like radio buttons ?:<br >
-        <label for="Yes">Yes</label>
-        <input type="radio" name="yes" value="yes"><br>
-        <label for="No">No</label>
-        <input type="radio" name="no"value="no"><br>
-        <label for="None of your business"> None of your business</label>
-        <input type="radio" name="?"value="?" ><br></label>
```

Comment:
For these controls, you WERE supposed to wrap the control in the label rather than using the for attribute.
All of these radio buttons need to have the same name in order to be in the same group